src/components/RecentSnippetItem.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+import { downloadSnippet } from '../helpers';
+
 const RecentSnippetItem = ({ snippet }) => {
   const snippetTitle = snippet.get('title') || `#${snippet.get('id')}, Untitled`;
+  const download = () => downloadSnippet(snippet);
```

Comment:
If we need functions maybe it's a good thing to make this class component like it is in `Snippet.jsx` to make things consistent?

Reply:
My understanding is that we need this functionality in two different places, so even if it was a class, we would need to call a separate function anyway (if we do not want to duplicate its code here), so I think this is fine.

Reply:
I didn't mean that we need to move this into separate component. What I meant is: rewrite RecentSnippetItem into class component so we will create download function in a way it is now in Snippet component. But it's not required.